addons/base_address_extended/models/res_partner.py

```diff
@@ -9,20 +9,19 @@
 
 class Partner(models.Model):
     _inherit = ['res.partner']
-    _name = 'res.partner'
 
-    street_name = fields.Char('Street Name', compute='_split_street',
-                              inverse='_set_street', store=True)
-    street_number = fields.Char('House', compute='_split_street', help="House Number",
-                                inverse='_set_street', store=True)
-    street_number2 = fields.Char('Door', compute='_split_street', help="Door Number",
-                                 inverse='_set_street', store=True)
+    street_name = fields.Char(
+        'Street Name', compute='_compute_street_data', inverse='_inverse_street_data', store=True)
+    street_number = fields.Char(
+        'House', compute='_compute_street_data', inverse='_inverse_street_data', store=True)
+    street_number2 = fields.Char(
+        'Door', compute='_compute_street_data', inverse='_inverse_street_data', store=True)
```

Comment:
Could we also rename the compute and inverse methods of "addons/base_address_extended/models/res_company.py " to the same things for consistency ?